api/Response.json

```
@@ -386,10 +386,10 @@
               }
             ],
             "safari": {
-              "version_added": false
+              "version_added": "11.1"
             },
             "safari_ios": {
-              "version_added": false
+              "version_added": "11"
```

GitHub lost my comment. Let's try this again.

Based on the commit date and the release date in `/browsers/safari_ios.json`, this probably shipped in the next release in our inventory:

```suggestion
              "version_added": "11.3"
```